src/main/resources/application.yaml

```diff
@@ -34,6 +34,13 @@ server:
 file:
   path: "files"
 
+ses:
+  accessKey: "AKIA4RSBG3WXTR7OUT3B"
```

Comment:
Provide placeholder for `accessKey` and `secretKey` like for database user and password